app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/alwayson/AlwaysOnLockDownDetector.kt

```diff
@@ -81,28 +85,41 @@ class AlwaysOnLockDownDetector @Inject constructor(
         removeNotification()
     }
 
-    private fun showNotification() {
-        val title = SpannableStringBuilder(resources.getString(R.string.atp_AlwaysOnLockDownNotificationTitle))
+    private suspend fun showNotification() {
+        val title = SpannableStringBuilder(getNotificationText())
+        val intent = getNotifIntent()
+
         val notification = DeviceShieldNotificationFactory.DeviceShieldNotification(title = title)
         deviceShieldAlertNotificationBuilder.buildAlwaysOnLockdownNotification(
             context,
             notification,
-            NotificationPressedHandler(),
+            intent,
         ).also {
             notificationManagerCompat.notify(notificationId, it)
         }
     }
 
-    private fun removeNotification() {
-        notificationManagerCompat.cancel(notificationId)
+    private suspend fun getNotificationText(): String {
+        val isAppTPEnabled = appTrackingProtection.isEnabled()
+        val isNetPEnabled = networkProtectionState.isEnabled()
+        return when {
+            isAppTPEnabled && isNetPEnabled -> R.string.vpn_LockdownNotificationTextWithNetPAndAppTPEnabled
+            isNetPEnabled -> R.string.vpn_LockdownNotificationTextWithNetPOnlyEnabled
+            else -> R.string.atp_AlwaysOnLockDownNotificationTitle
+        }.run {
+            resources.getString(this)
+        }
     }
 
-    private class NotificationPressedHandler constructor() : ResultReceiver(Handler(Looper.getMainLooper())) {
-        override fun onReceiveResult(
-            resultCode: Int,
-            resultData: Bundle?,
-        ) {
-            logcat { "Lockdown notification pressed" }
+    private suspend fun getNotifIntent(): Intent {
+        val isNetPEnabled = networkProtectionState.isEnabled()
+        return when {
+            isNetPEnabled -> globalActivityStarter.startIntent(context, NetworkProtectionManagementScreenNoParams)!!
+            else -> DeviceShieldTrackerActivity.intent(context = context)
```

Comment:
Nit: why not the following:
```suggestion
            else -> globalActivityStarter.startIntent(context, AppTrackerActivityWithEmptyParams)
```